slu-status/test/app.e2e-spec.ts

```diff
@@ -71,8 +90,8 @@ describe('AppController (e2e)', () => {
 			const { status, body } = await request(app.getHttpServer())
 				.get('/api/v1/status/did:iota:1234')
 				.set('X-API-KEY', '2b3fe07d-b7db-49cb-8300-d32139e3d435');
-			expect(status).toBe(200);
-			expect(body).toEqual({});
+			expect(status).toBe(500);
+			expect(body.message).toBe('Could not find document in the collection.');
 		});
 
 		it('/:id (GET): wrong api-key', async () => {
```

Comment:
Is there some code missing? Where body is defined?

Reply:
body is defined in line 131:
 const { status, body } = await request(app.getHttpServer())